doc/steps.md

```diff
@@ -0,0 +1,503 @@
+# Role Variables Descriptions
```

Comment:
Why did I open `steps` doc and see  `Role Variables Descriptions`?

I suggest changing the file structure:

# Role steps

Here are described steps that can be combined in the role [scenarios](/doc/scenario.md).
Each step description says what variables are required for the step and what facts are set by step.

## Role Steps List

... list of steps...

## Variables set on the role preparation

Some useful variables are always established during the role preparation. It can be used by any step:

lalala, instance_info, lalala

## Role Steps Description

...

(probably steps should have `###` level)

...